gkeys/gkeys/actions.py

```diff
@@ -127,7 +129,21 @@ def updateseed(self, args):
                 success = False
             else:
                 print("Update succeeded.\n")
-            messages = fetch_messages + ["Update operation:"]  + [install_messages]
+            messages = fetch_messages + [" Update operation:"] + [install_messages]
+            success, new_gkeys = self.listseed(args)
+            added_gkeys, changed_gkeys  = self.seedhandler.compare_seeds(old_gkeys, new_gkeys)
+            print("Updated revoked GKeys:")
```

Comment:
this doesn't read quite right.

I think drop the revoked word here, make it just "Updated keys", it is not depended on them being revoked
revoked should be sub list probably.